### PR TITLE
Updated CODEOWNERS to reflect the surviving TOC

### PR DESCRIPTION
**Pull request by nrjpoddar**

### CODE REVIEW

**Review by smawson — APPROVED** [review]
Survival of the fittest?

File: CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1,4 @@
-*                @linsun @dcberg @louiscryan @smawson @duderino @nrjpoddar
+*                @linsun @louiscryan @smawson @nrjpoddar
```
Comment (brian-avery) [inline CODEOWNERS:1]: Out of curiosity, why don't we use `istio/technical-oversight-committee` in codeowners?

**Comment by nrjpoddar** [comment]
> Survival of the fittest?

I hope that's the case ;)

File: CODEOWNERS (hunk 1 above)
Comment (nrjpoddar) [reply CODEOWNERS:1]: Good question. I don't know, not an expert on all of this but I would happily take recommendation from someone like you who is. We can also do some trials as follow-on PRs?

**Comment by duderino** [comment]
A small sob was heard in the distance... around the vicinity of the great america amusement park

File: CODEOWNERS (hunk 1 above)
Comment (linsun) [reply CODEOWNERS:1]: I think we have individual listed here to ensure the approval from at least one TOC on api repo.  group didn't work when we tried it 2 years ago

**Review by linsun — APPROVED** [review]

**Pull request merged**